tasks/openshift-ipi-registry-configure/0.1/openshift-ipi-registry-configure.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: openshift-ipi-registry-configure-v0-1
+spec:
+  description: |-
+    Task to configure the OpenShift registry to make the initial IPI deployment complete successfully.
+  params:
+  - name: IMAGE
+    type: string
+    default: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+    description: |-
+      Container image to run this stage.
+      Examples of valid values include:
+        image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+      Note the service account used to execute this task must have a secret with appropriate permissions to pull the specified image.
+  steps:
+  - name: wait-for-operator
+    image: $(params.IMAGE)
+    script: |
+      export KUBECONFIG=$(workspaces.persistent.path)/auth/kubeconfig
```

Comment:
Could we use the env parameter to define this instead of having it in the script?